### DISCUSSION TITLE
Mastodon won’t start after upgrade Ubuntu from 18.04 LTS to 22.04 LTS

### DISCUSSION THREAD

**Discussion by LolYangccool** [mastodon/mastodon#26366] [General]
Hello,
I have the same bug as here: https://github.com/mastodon/mastodon/issues/19934
I have try to reinstall Mastodon on the existing setup but it doesn’t work.

Can you help me ?
On 18.04 image upload don’t work because of a bug on ImageMagick I think.

Thank you.

**Comment by BossMAN559** [comment]
do you get an access denied in your nginx error log?
if so try this command
chmod a+x /home/mastodon/

**Comment by LolYangccool** [comment]
My problem was fixed earlier this day.
Thanks you.
